config/config.go

```diff
@@ -0,0 +1,10 @@
+package config
+
+type Config struct {
+	Github
```

Comment:
Also, add a config for the upstream address here:
```suggestion
	ParserServiceAddress string `env:"PARSER_SERVICE_ADDRESS,required"`
	Github
```